src/test/playwright/support/pageobjects/exercises/quiz/QuizExerciseCreationPage.ts

```diff
@@ -46,17 +46,23 @@ export class QuizExerciseCreationPage {
         await this.page.locator('#drag-and-drop-question-title').fill(title);
 
         await this.uploadDragAndDropBackground();
-        await this.page.mouse.move(50, 50);
+        const element = await this.page.$('.background-area');
```

Comment:
```suggestion
        const element = await this.page.locator('.background-area');
```

Locator-based methods are preferred